### PR TITLE
check whatever Bukkit#getWorld is null

### LINKED ISSUE #243: Unloading and deleting world directory causes server crash
I have the following code that unloads the world after it's used:

```
File f = new File(world.getName() + "/");
boolean unloaded = Bukkit.unloadWorld(world, false);
Bukkit.broadcastMessage("unloaded: " + unloaded); // Prints true

if (f.isDirectory()) {
   try {
      FileUtils.deleteDirectory(f);
   } catch (IOException e) {
      e.printStackTrace();         
   }
}
```

but it spams this until it crashes:
https://pastebin.com/raw/ueUQ6aMd

Is unloading worlds supported?

### PR DESCRIPTION
**Pull request by ham1255**
fixes #243

### CODE REVIEW

File: patches/server/0181-check-whatever-Bukkit-getWorld-is-null.patch [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+diff --git a/src/main/java/puregero/multipaper/MultiPaper.java b/src/main/java/puregero/multipaper/MultiPaper.java
+index 0428f7d8702c8f1e837a9a95d31dc501374880bf..aa42f350731d6d6aaab76047733867d9f2efe1b9 100644
+--- a/src/main/java/puregero/multipaper/MultiPaper.java
++++ b/src/main/java/puregero/multipaper/MultiPaper.java
+@@ -147,7 +147,7 @@ public class MultiPaper {
+                 level.saveIncrementally(true);
+             }
+         }
+-
```
Comment (PureGero) [inline patches/server/0181-check-whatever-Bukkit-getWorld-is-null.patch:15]: Please remove unintended formatting changes

**Review by PureGero — CHANGES_REQUESTED** [review]

File: patches/server/0181-check-whatever-Bukkit-getWorld-is-null.patch (hunk 1 above)
Comment (ham1255) [reply patches/server/0181-check-whatever-Bukkit-getWorld-is-null.patch:15]: not sure where these are comming from didn't add single lines.

Comment (ham1255) [reply patches/server/0181-check-whatever-Bukkit-getWorld-is-null.patch]: fixed the format

File: patches/server/0181-check-whatever-Bukkit-getWorld-is-null.patch [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+             update.runUpdates((pos, state) -> {
+diff --git a/src/main/java/puregero/multipaper/commands/MPMapCommand.java b/src/main/java/puregero/multipaper/commands/MPMapCommand.java
+index 913325e43b9501809f3973987f120d8ef06c2d79..7a0181d350a5a72f8bd85a8456d3fef4f1a40af9 100644
+--- a/src/main/java/puregero/multipaper/commands/MPMapCommand.java
++++ b/src/main/java/puregero/multipaper/commands/MPMapCommand.java
+@@ -46,7 +46,6 @@ public class MPMapCommand extends Command {
+ 
+     private LevelChunk getChunk(Player player, int x, int z) {
+         ChunkAccess chunkAccess = MultiPaper.getChunkAccess(player.getWorld().getName(), ((CraftPlayer) player).getHandle().blockPosition().offset(x << 4, 0, z << 4));
+-
```
Comment (PureGero) [inline patches/server/0181-check-whatever-Bukkit-getWorld-is-null.patch:74]: One more unintended format change here

**Review by PureGero — CHANGES_REQUESTED** [review]

File: patches/server/0181-check-whatever-Bukkit-getWorld-is-null.patch (hunk 2 above)
Comment (ham1255) [reply patches/server/0181-check-whatever-Bukkit-getWorld-is-null.patch]: Fixed it

**Comment by tomaica1934** [comment]
merge this pls

**Comment by ham1255** [comment]
i just removed creation of new patch and modified existing patches.

**Comment by PureGero** [comment]
Requires rebase, then will be merged

**Pull request merged**